pybliometrics/scopus/abstract_retrieval.py

```diff
@@ -747,29 +747,58 @@ def __str__(self):
         Assumes the document is a journal article and was loaded with
         view="META_ABS" or view="FULL".
         """
-        date = self.get_cache_file_mdate().split()[0]
-        # Authors
-        if self.authors:
-            if len(self.authors) > 1:
-                authors = _list_authors(self.authors)
+        if self._view in ('FULL', 'META_ABS', 'META'):
+            date = self.get_cache_file_mdate().split()[0]
+            # Authors
+            if self.authors:
+                if len(self.authors) > 1:
+                    authors = _list_authors(self.authors)
+                else:
+                    a = self.authors[0]
+                    authors = str(a.given_name) + ' ' + str(a.surname)
```

Comment:
let's rewrite this a bit: 
`authors = " ".join([a.given_name or "", a.surname or ""])`